tools/executors/prepare-library/utils/sync-versions.ts

```diff
@@ -1,10 +1,12 @@
 import { logger, ProjectConfiguration } from '@nrwl/devkit';
-import { readdirSync, readFileSync, writeFileSync } from 'fs-extra';
+import { readFileSync, writeFileSync } from 'fs-extra';
 import { major } from 'semver';
 import { PrepareOptions } from './prepare.options';
+import { glob } from 'glob';
 
 const packageJson = JSON.parse(readFileSync(`./package.json`, 'utf8'));
-const excludedFileTypes = ['js', 'mjs', 'map', 'ts'];
+const excludedFilesPatterns = ['md', 'mjs', 'map', 'ts'].map((fileType) => `**/*.${fileType}`);
```

Comment:
in compiled dist
<img width="700" alt="image" src="https://user-images.githubusercontent.com/20265336/193783298-74e9fc84-0239-426c-a148-728ebb066e8a.png">

Reply:
It reports this:

=== Copying schematics of core ===
✅ Copied schematics for project core
=== Syncing versions in core ===
✅ Replaced "VERSION_PLACEHOLDER" with "0.36.1-rc.23" in dist/libs/core/package.json
✅ Replaced "VERSION_PLACEHOLDER" with "0.36.1-rc.23" in dist/libs/core/package.json
✅ Replaced "ANGULAR_VER_PLACEHOLDER" with "^14.0.0" in dist/libs/core/package.json
✅ Replaced "ANGULAR_VER_PLACEHOLDER" with "^14.0.0" in dist/libs/core/package.json
✅ Replaced "ANGULAR_VER_PLACEHOLDER" with "^14.0.0" in dist/libs/core/package.json
✅ Replaced "ANGULAR_VER_PLACEHOLDER" with "^14.0.0" in dist/libs/core/package.json
✅ Replaced "ANGULAR_VER_PLACEHOLDER" with "^14.0.0" in dist/libs/core/package.json
✅ Replaced "ANGULAR_VER_PLACEHOLDER" with "^14.0.0" in dist/libs/core/package.json
✅ Replaced "RXJS_VER_PLACEHOLDER" with "7.5.6" in dist/libs/core/package.json
✅ Replaced "FAST_DEEP_EQUAL_VER_PLACEHOLDER" with "3.1.3" in dist/libs/core/package.json
✅ Replaced "FDSTYLES_VER_PLACEHOLDER" with "0.25.0-rc.23" in dist/libs/core/package.json
✅ Replaced "FOCUSTRAP_VER_PLACEHOLDER" with "6.7.3" in dist/libs/core/package.json
✅ Replaced "LODASH_ES_VER_PLACEHOLDER" with "4.17.21" in dist/libs/core/package.json
✅ Replaced "COMPARE_VERSIONS_VER_PLACEHOLDER" with "4.1.3" in dist/libs/core/package.json
✅ Replaced "THEMING_VER_PLACEHOLDER" with "11.1.40" in dist/libs/core/package.json
✅ Replaced "FDSTYLES_VER_PLACEHOLDER" with "0.25.0-rc.23" in dist/libs/core/schematics/ng-add/index.js
✅ Replaced "FDSTYLES_VER_PLACEHOLDER" with "0.25.0-rc.23" in dist/libs/core/schematics/ng-add/index.js
✅ Replaced "THEMING_VER_PLACEHOLDER" with "11.1.40" in dist/libs/core/schematics/ng-add/index.js
✅ Replaced "THEMING_VER_PLACEHOLDER" with "11.1.40" in dist/libs/core/schematics/ng-add/index.js

And I checked output, it is correct, can not find that anywhere. Did you just launch `build-umbrella`?